services/src/modules/plugins/logging.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ApolloServerPlugin,
+  GraphQLRequestContext,
+  GraphQLRequestContextWillSendResponse,
+} from 'apollo-server-plugin-base';
+import { FastifyInstance } from 'fastify';
+import { Counter, Histogram } from 'prom-client';
+import logger from '../logger';
+
+let errorsInResponseCounter: Counter<string>;
+let requestDurationHistogram: Histogram<string>;
+
+export function createLoggingPlugin(fastifyInstance: FastifyInstance): ApolloServerPlugin {
```

Comment:
Maybe we can separate logging from metrics, either via different plugins or just more separation inside the function? Because right now `createLoggingPlugin`'s code is like 90% metrics

Reply:
I considered it. I did both thing in one plugin because there is no a lot of code. But you're right. I'll separate them